stable/yugabyte/templates/service.yaml

```diff
@@ -99,6 +99,7 @@ spec:
   selector:
     {{- include "yugabyte.appselector" ($appLabelArgs) | indent 4 }}
   type: {{ $endpoint.type }}
+  loadBalancerIP: {{ $endpoint.loadBalancerIP }}
```

Comment:
Does this affect upgrades? For example, if it is not set in a previous version, then we upgrade, will the upgrade succeed?

To protect against accidental chart changes, we could modify this to only specify `loadBalancerIP` if the `$endpoint.loadBalancerIP` value is set.

Reply:
> To protect against accidental chart changes, we could modify this to only specify loadBalancerIP if the $endpoint.loadBalancerIP value is set.

Yeah, Make sense.